cmd/influxd/inspect/delete_tsm/delete_tsm.go

```diff
@@ -141,9 +142,12 @@ func (a *args) process(cmd *cobra.Command, path string) error {
 	if err := w.Close(); err != nil {
 		return fmt.Errorf("failed to close TSM Writer: %w", err)
 	}
+	if err := r.Close(); err != nil {
+		return fmt.Errorf("failed to close TSM Reader: %w", err)
+	}
 
 	// Replace original file with new file.
-	if err := os.Rename(outputPath, path); err != nil {
+	if err := file.RenameFile(outputPath, path); err != nil {
```

Comment:
Existing helper to first delete `path` if it's there, then move `outputPath` -> `path`